### DISCUSSION TITLE
Can I add a Vite example?

### DISCUSSION THREAD

**Discussion by Dhaiwat10** [rainbow-me/rainbowkit#552] [Ideas]
Thought we should show some love to the Vite community and add a Vite example for Rainbowkit. Can I start working on a PR?

**Comment by markdalgleish** [comment]
Sorry, I missed your question about working on a PR! We just added a Vite example recently: https://github.com/rainbow-me/rainbowkit/pull/581

If there's any way we can make the example better, please let us know 🙏
